src/components/cart/CartTotal/CartTotal.tsx

```diff
@@ -1,68 +1,123 @@
 import { styled } from 'styled-components';
 import Spacer from '../../common/Spacer/Spacer';
 import { formatPrice } from '../../../utils/formatPrice';
+import OrderConfirmModal from '../../order/OrderConfirmModal/OrderConfirmModal';
+import { removeComma } from '../../../utils/removeComma';
+import { CloseIcon } from '../../../assets/svg';
+import useCartTotal from './useCartTotal';
+import { FREE_SHIPPING_PRICE, SHIPPING_FEE } from '../../../constants/cart';
+import useModal from '../../common/Modal/useModal';
+import type { CartItem } from '../../../types/cart';
+import useCloseModalOnUnmount from '../../common/Modal/useCloseModalOnUnmount';
 
-const FREE_SHIPPING_PRICE = 30_000;
-const SHIPPING_FEE = 3_000;
+interface CartTotalProps {
+  selectedCartItemIds: Set<CartItem['id']>;
+  totalProductPrice: number;
+}
 
-const calcTotalOrderPrice = (
-  totalProductPrice: number,
-  isFreeShipping: boolean,
-) => {
-  if (totalProductPrice <= 0) return 0;
-
-  return isFreeShipping ? totalProductPrice : totalProductPrice + SHIPPING_FEE;
-};
-
-const CartTotal = ({ totalProductPrice }: { totalProductPrice: number }) => {
+const CartTotal = ({
+  selectedCartItemIds,
+  totalProductPrice,
+}: CartTotalProps) => {
+  const {
+    point,
+    usingPoint,
+    updateUsingPoint,
+    clearPoint,
+    useAllPoint,
+    totalOrderPrice,
+    totalPaymentPrice,
+  } = useCartTotal(totalProductPrice);
```

Comment:
용도가 다르다면 export 할 때 묶어서 내보내고 사용하는 곳에서 아래처럼 사용하도록 구분해주는 방법도 있을거 같네요~ 

```suggestion
  const {
    point,
    usingPoint,

    handlers: { 
      updateUsingPoint,
      clearPoint,
      useAllPoint,
    }
    
    infos: {
      totalOrderPrice,
      totalPaymentPrice,    
    }
  } = useCartTotal(totalProductPrice);
```